### DISCUSSION TITLE
Vote to nominate @joshuatcasey to be a Dotnet Core contributor

### DISCUSSION THREAD

**Discussion by ForestEckhardt** [paketo-buildpacks/feedback#117] [.NET Core Team]
I'd like to nominate @joshuatcasey as a contributor for the Dotnet Core subteam. He has made large contributions of his own to the space (see [dotnet-execute PR](https://github.com/paketo-buildpacks/dotnet-execute/pull/277)) and has many other PRs in the works.

👍 / 👎 ?

cc: @paketo-buildpacks/dotnet-core-maintainers

**Comment by ryanmoran** [comment]
Congrats @joshuatcasey! You are now a .NET Core Contributor.
